relate/settings.py

```diff
@@ -164,6 +164,8 @@
 
 RELATE_ADMIN_EMAIL_LOCALE = "en_US"
 
+RELATE_SESSION_RESTART_COOLDOWN_SECONDS = 10
```

Comment:
Because we're using `getattr()` with a default, this isn't needed. I've gone ahead and removed it.